sound/soc/sof/ipc.c

```diff
@@ -32,6 +32,9 @@ static void ipc_stream_message(struct snd_sof_dev *sdev, u32 msg_cmd);
 struct snd_sof_ipc {
 	struct snd_sof_dev *sdev;
 
+	/* Synchronise access to HIPC* registers */
```

Comment:
HIPC -> IPC registers to make this less Intel-centric?

I am also wondering if we could take the scissor and make this the snd_sof_ipc_doorbell structure, others may have different ways of sending stuff across to the firmware

Reply:
I've reworded the comment. As for the renaming - sure, we can do that, but maybe after we've fixed all the main upstreaming-blockers?